src/net/sourceforge/kolmafia/session/InventoryManager.java

```diff
@@ -808,6 +809,8 @@ private static String doRetrieveItem(
         RequestThread.postRequest(new HermitRequest(itemId, retrieveCount));
       }
 
+      RequestThread.postRequest(new HermitRequest(itemId, missingCount));
+
```

Comment:
This is no longer correct.

Reply:
This is a vestige of the merge I did you you - which retained my version of the needed change and removed (all but those lines) of yours.

Mine was correct, because you really do have to limit the number of 11-leaf clovers you get from the Hermit to however many he (still) has on offer today. Your change eliminated that check.

So, please remove the extra HermitRequest. Thanks.